package.json

```diff
@@ -22,11 +23,18 @@
     "gulp-shell": "^0.8.0",
     "gulp-terser": "^2.0.1",
     "gulp-uglify": "^3.0.2",
-    "jsdoc": "^3.6.7",
+    "jsdoc": "^3.6.10",
     "jshint": "^2.13.1"
   },
   "scripts": {
     "build": "gulp",
+    "types": "jsdoc -t node_modules/tsd-jsdoc/dist -r 3Dmol/. -d types",
+    "test": "npm run glcheck",
```

Comment:
Will the fact that both of us have something under "scripts" : { "test" : ...} be a problem? It seems like this will create a merge conflict but let me know if I'm wrong

Reply:
This will cause a merge conflict but is up to the pr that merges 2nd to resolve the conflict